config/environments/production.rb

```diff
@@ -99,7 +99,8 @@
       user_name:            ENV.fetch("SMTP_USER"),
       password:             ENV.fetch("SMTP_PASS"),
       authentication:       ENV.fetch("SMTP_AUTHENTICATION"),
-      enable_starttls_auto: ENV.fetch("SMTP_TLS").present?
+      enable_starttls_auto: ENV.fetch("SMTP_STARTTLS").present?,
+      ssl:                  ENV.fetch("SMTP_SSL").present?
```

Comment:
Dans la documentation rails : [Action Mailer Configuration](https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration)
> `:ssl/:tls` - Enables the SMTP connection to use SMTP/TLS (SMTPS: SMTP over direct TLS connection)

De ce que j'en comprends, c'est que les options `:ssl` et `:tls` sont des alias pour activer TLS pour toutes les connexions SMTP. Comme le suggère @Dan33l, le nom de la variable d'environnement  `SMTP_TLS` est peut-être  plus pertinent que `SMTP_SSL`.

```suggestion
      tls:                  ENV.fetch("SMTP_TLS").present?
```

Ce qui donnera au final : 
```diff
-       enable_starttls_auto: ENV.fetch("SMTP_TLS").present?,
+       enable_starttls_auto: ENV.fetch("SMTP_STARTTLS").present?,
+       tls:                  ENV.fetch("SMTP_TLS").present?
```

C'est un breaking change pour les instances qui utiliserai `STARTTLS`, mais de mémoire cela ne concernait qu'une future instance DS qui n'a pas vu le jour.